packages/venia-concept/src/components/MiniCart/kebab.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+
+import Icon from 'src/components/Icon';
+import classify from 'src/classify';
+import defaultClasses from './kebab.css';
+
+class Kebab extends Component {
+    render() {
+        const { classes, isOpen, onFocus, onBlur, children } = this.props;
+        const toggleClass = isOpen
+            ? classes.dropdown + ' ' + classes.active
```

Comment:
Rather than concatenating classnames, try to use composition inside the CSS module.

```css
.dropdown {
  transform: scale(0);
}

.dropdown_active {
  composes: root; /* extends `.dropdown` above */
  transform: scale(1);
}
```

```es6
const className = isOpen ? 'dropdown_active' : 'dropdown'
```